Tests/SwiftDocCTests/Rendering/HeadingAnchorTests.swift

```diff
@@ -0,0 +1,59 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021-2024 Apple Inc. and the Swift project authors
```

Comment:
nit: This is a new file
```suggestion
 Copyright (c) 2024 Apple Inc. and the Swift project authors
```

Reply:
Fixed 👌🏻